server/request_intercepts.go

```diff
@@ -150,3 +152,48 @@ func (r *RpcRequest) intercept_eth_call_to_FlashRPC_Contract() (requestFinished
 	r.logger.Info("Intercepted eth_call to FlashRPC contract")
 	return true
 }
+
+func (r *RpcRequest) intercept_signed_eth_getTransactionCount() (requestFinished bool) {
+	signingAddress, err := flashbots.ParseSignature(r.flashbotsSignature, r.flashbotsSignatureBody)
+	if errors.Is(err, flashbots.ErrNoSignature) {
+		r.logger.Info("[eth_getTransactionCount] No signature found")
+		return false
+	}
+
```

Comment:
Oh wow just realized when refactoring this isn't handling the else case and should be returning an error result.  The refactor I'm doing to move the signature check earlier in the flow helps make that more obvious 😊

Reply:
Fixed in https://github.com/flashbots/rpc-endpoint/pull/151/commits/d90fb9b1c8de38358dbf4e5c748e52b234462300.